src/qt/netwatch.h

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2017 The Bitcoin Core developers
```

Comment:
nit 2018.

Reply:
2018 is wrong. This code hasn't been substantially changed since 2017.